workspaces/ui-v2/src/components/Page/Page.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { FC, useEffect } from 'react';
```

Comment:
@JaapRood Revived your page component - we currently have a lot of usage with PageLayout, but I think this should take over those adhoc pagelayout components, etc